matlabCode/createBIDS_anat_Full_T1w_json.m

```diff
@@ -241,11 +245,20 @@
 
 
 %% Write JSON
-json_options.indent = '    '; % this makes the json look pretier when opened in a txt editor
+% this makes the json look pretier when opened in a txt editor
```

Comment:
pret**t**ier.
(this option is specific to JSONio and will not work with jsonencode).

Reply:
I think that for now we will stick with JSONio so this should be fine, unless we have some very good reason not to.